ppl/lake/ztests/s3/s3root.yaml

```diff
@@ -23,9 +23,7 @@ outputs:
       [1587518620062237300-1587513611063914690]
       [1587513592062544400-1587508830068523240]
       ===
-      #0:record[count:uint64]
-      0:[1000;]
-      #0:record[key:int64,count:uint64,first:time,last:time]
-      0:[2;1;1587518620.0622373;1587513611.06391469;]
+      {count:1000 (uint64)} (=0)
+      {key:2,count:1 (uint64),first:2020-04-22T01:23:40.0622373Z,last:2020-04-22T00:00:11.06391469Z} (=0)
```

Comment:
Similar observation to the `map.yaml` one above: The `=0` type definition is re-used.

Reply:
Yes, this is ok too.